semeio/workflows/misfit_preprocessor/config.py

```diff
@@ -251,51 +252,40 @@ def _inject_default_t(config):
             cs.MetaKeys.ElementValidators: (_one_of(SPEARMAN_CORRELATION, AUTO_SCALE),),
             cs.MetaKeys.Default: AUTO_SCALE,
         },
-        SPEARMAN_CORRELATION: SPEARMAN_CORRELATION_SCHEMA,
-        AUTO_SCALE: AUTO_SCALE_SCHEMA,
+        "clustering": HIERARCHICAL_SCHEMA,
+        "pca": PCA_SCHEMA,
     },
 }
 
 
-_SCALING_SCHEMA = {
+@cs.transformation_msg("Detecting method.")
+def insert_method(layer):
```

Comment:
Should probably prefix this with `_` and/or add a unit test.

Reply:
It might also be good with a docstring in this function to make it clean what the intention is.

Reply:
In the first comment, I have to think about a unit test.

Reply:
We expanded `test_config_workflow_invalid` in `test_config.py` for this.